include/aikido/io/util.hpp

```diff
@@ -0,0 +1,22 @@
+#ifndef AIKIDO_IO_UTIL_HPP_
+#define AIKIDO_IO_UTIL_HPP_
+
+#include <aikido/io/CatkinResourceRetriever.hpp>
+#include <dart/dart.hpp>
+#include <dart/utils/urdf/DartLoader.hpp>
+
+using dart::dynamics::SkeletonPtr;
+
+namespace aikido {
+namespace io {
+
+const SkeletonPtr makeBodyFromURDF(
+    const std::shared_ptr<aikido::io::CatkinResourceRetriever>
```

Comment:
Maybe this can be a `dart::common::ResourceRetrieverPtr`, since that's what the DART URDF loader is expecting.